be/src/vec/data_types/data_type_date.h

```diff
@@ -60,6 +60,12 @@ class DataTypeDate final : public DataTypeNumberBase<Int64> {
     bool equals(const IDataType& rhs) const override;
     std::string to_string(const IColumn& column, size_t row_num) const override;
     void to_string(const IColumn& column, size_t row_num, BufferWritable& ostr) const override;
+    std::string to_string(Int64 int_val) const {
+        doris::VecDateTimeValue value = binary_cast<Int64, doris::VecDateTimeValue>(int_val);
+        char buf[64];
```

Comment:
why 64?

Reply:
DataTypeDate is actually DataTypeNumberBase<Int64>

Reply:
so may be 22 bytes is enough? `INT64_MIN` is `-9223372036854775808`

Reply:
it's formated to datetime format, e.g. 2024-05-16 00:00:00.999999，use 64 to be large enough to contain the result string.